src/components/policy/policy_regular/src/policy_table/enums.cc

```diff
@@ -727,6 +727,8 @@ bool IsValidEnum(FunctionID val) {
       return true;
     case DeleteSubMenuID:
       return true;
+    case ShowAppMenuID:
```

Comment:
@mkorniichuk please notice that this file is generated, so do not edit it
https://github.com/smartdevicelink/sdl_core/blob/develop/src/components/policy/policy_regular/src/policy_table/enums.cc#L1

Reply:
@LitvinenkoIra fixed within 176e6bb7cadd8fc58016f78bdfe23b41cd7f2213